curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -73,6 +71,12 @@ const labelElem = document.querySelector('label[for="loving"]');
 assert(labelElem.textContent.replace(/\s/g, '').match(/Loving/i));
 ```
 
+There should be a space between your checkbox and your new `label` element.
+
+```js
+assert.match(code, /<input id=('|")loving\1 type=('|")checkbox\2> <label for=('|")loving\3>Loving<\/label>/);
```

Comment:
Adding new ones or changing existing tests in challenges should be done across all localizations (ex. de, it, ua, etc.)